src/plugins/audio-stream/backend.ts

```
@@ -0,0 +1,137 @@
+  return comments.length ? comments : ['TITLE=Pear Desktop'];
+}
+
+export const backend = createBackend<
+  {
+    app: Hono;
+    server?: ServerType;
+  },
+  AudioStreamConfig
+>({
+  app: new Hono()
+    .get('/stream', (ctx) => {
```

ESLint

[error] Delete `⏎····`

(prettier/prettier)